internal/component/azure/servicebus/subscription.go

```diff
@@ -315,16 +320,20 @@ func (s *Subscription) CompleteMessage(ctx context.Context, m *azservicebus.Rece
 	}
 }
 
-func (s *Subscription) addActiveMessage(m *azservicebus.ReceivedMessage) {
-	s.logger.Debugf("Adding message %s to active messages on %s", m.MessageID, s.entity)
+func (s *Subscription) addActiveMessage(m *azservicebus.ReceivedMessage) error {
+	if m.SequenceNumber == nil {
```

Comment:
From https://docs.microsoft.com/en-us/azure/service-bus-messaging/message-sequencing it seems that the sequence number will always be present. Is that correct?
```

Sequencing and timestamping are two features that are always enabled on all Service Bus entities and surface through the SequenceNumber and EnqueuedTimeUtc properties of received or browsed messages.
```

Reply:
Yes sequence number should always be present. The only reason I do a nil check is to avoid a panic if there is a bug in the underlying SDK when deserializing the message.

Reply:
Updated with the behaviour mentioned above